app/services/food_service.rb

```diff
@@ -0,0 +1,10 @@
+class FoodService
+  def self.conn
+    Faraday.new(url: "https://api.nal.usda.gov")
+  end
+
+  def self.get_foods_by_type(food_type)
+    response = conn.get("/fdc/v1/foods/search?query=#{food_type}&pageSize=50&api_key=#{ENV["food_api_key"]}")
```

Comment:
This imposes a `pageSize` on every API call. It would be more conventional to make this param dynamic:
```
def self.get_foods_by_type(food_type, page_size = 10)
  response = conn.get("/fdc/v1/foods/search?query=#{food_type}&pageSize=#{page_size}&api_key=#{ENV["food_api_key"]}")
  data = JSON.parse(response.body, symbolize_names: true)
end
```
In this case, though, I might suggest returning all results and having the facade handle the limiting logic instead. That way, we could get a count of total results and the 10 foods to show, all with one API call. I'll leave a comment in the facade with an example of what that might look like.